src/main/java/teammates/logic/backdoor/BackDoorServlet.java

```diff
@@ -133,149 +52,145 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
         resp.flushBuffer();
     }
 
-    private String executeBackendAction(HttpServletRequest req, String action)
-            throws EntityDoesNotExistException, InvalidParametersException, IOException {
-        // TODO: reorder in alphabetical order
+    @SuppressWarnings("PMD.SwitchStmtsShouldHaveDefault") // no default so that each case is accounted for
```

Comment:
Is the "switch statements should have default" a default rule? It seems odd because the fact that we say ```no default so that each case is accounted for``` and that is something that I also believe it, why is it on?